plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/TestUnityMetastore.java

```diff
@@ -45,7 +45,7 @@ void test()
                 .setHttpBearerToken(databricksToken)
                 .setAdditionalHeaders("X-Databricks-Catalog-Name:" + databricksCatalogName);
         ThriftMetastoreClient client = ((ThriftMetastoreClientFactory) new HttpThriftMetastoreClientFactory(config, new TestingNodeManager(), OpenTelemetry.noop()))
-                .create(metastoreUri, Optional.empty());
+                .create(metastoreUri, Optional.empty(), Optional.of("hive"));
```

Comment:
HttpThriftMetastoreClientFactory should not need the catalog name as parameter or it should be `Optional.empty()`